README.md

```diff
@@ -110,7 +110,7 @@ See [remarkable.guide](https://remarkable.guide/tech/factory-reset.html) for mor
 ## Cloud Tools
 
 - [CUPS Printing](https://ofosos.org/2018/10/22/printing-to-remarkable-cloud-from-cups/) - Script to print directly to reMarkable Cloud from CUPS using rMAPI.
-- [kindle2pdf](https://github.com/teticio/kindle2pdf) - Convert owned Kindle books to PDF without needing a Kindle device. Includes `pdf2remarkable` script to upload PDFs to the reMarkable Cloud.
+- [pdf2remarkable](https://github.com/teticio/pdf2remarkable) - Script to upload PDFs to the reMarkable Cloud.
```

Comment:
https://github.com/reHackable/awesome-reMarkable/blob/master/CONTRIBUTING.md
Please move this line to be in alphabetical order

Reply:
Didn't think of that, beg your pardon. Will do.